src/app/page.js

```diff
@@ -1,11 +1,15 @@
 "use client";
 
 import styled from "styled-components";
+import Navbar from "../components/Navbar";
 
 const Root = styled.div`
```

Comment:
Now that you know the concept of `layout.js` you can probably use it instead of using this style on `page.js`